polyphemus/db/migrations/002_create_cursors.rb

```diff
@@ -0,0 +1,14 @@
+Sequel.migration do 
```

Comment:
Cursors keep the state of ETL jobs so that only newly updated files are actually searched and processed.